constellations/gemini-admin/src/home/home.js

```diff
@@ -1,205 +1,110 @@
-        let achieves = this.state.achieves.map((row, index) => {
-            return <DashboardRow key={index} title={row.message} />;
-        });
-        this.setState({
-            noUnpub:
-                achieves.length == 0
-                    ? "All items are published! Your website is up to date!"
-                    : "",
-            rows: achieves,
-            selection: "achievements",
-            switch: achieves.length != 0,
-        });
-    }
-
-    onClickSemesters() {
-        let semesters = this.state.semesters.map((row, index) => {
-            return <DashboardRow key={index} title={row.semesterId} />;
-        });
-        this.setState({
-            noUnpub:
-                semesters.length == 0
-                    ? "All items are published! Your website is up to date!"
-                    : "",
-            rows: semesters,
-            selection: "semesters",
-            switch: semesters.length != 0,
-        });
-    }
-
-    onClickSessions() {
-        let sessions = this.state.sessions.map((row, index) => {
-            return <DashboardRow key={index} title={row.sessionId} />;
-        });
-        this.setState({
-            noUnpub:
-                sessions.length == 0
-                    ? "All items are published! Your website is up to date!"
-                    : "",
-            rows: sessions,
-            selection: "sessions",
-            switch: sessions.length != 0,
+    render() {
+        let unpublishedClasses = this.state.classes.map((row, index) => {
+            return <li key={index}> {row.classId} </li>;
         });
-    }
 
-    onClickPage() {
-        window.location.hash = this.state.selection;
-        console.log("Go to Page clicked");
-    }
-
-    onClickPublishAll() {
-        console.log("Publish All!");
-    }
-
-    render() {
-        let unpubMessage = <div> </div>;
-        if (this.state.switch) {
-            unpubMessage = (
-                <h4>
-                    {" "}
-                    You have {this.state.rows.length} unpublished{" "}
-                    {this.state.selection}{" "}
-                </h4>
-            );
-        }
         return (
             <div id="view-home">
-                <div className="home-left">
-                    <h2> Unpublished Content </h2>
-                    <ul>
-                        <button onClick={this.onClickPrograms}>Programs</button>
-                        <button onClick={this.onClickClasses}>Classes</button>
-                        <button onClick={this.onClickLocations}>
-                            Locations
-                        </button>
-                        <button onClick={this.onClickAchievements}>
-                            Achievements
-                        </button>
-                        <button onClick={this.onClickSemesters}>
-                            Semesters
+                <h1>Administrator Dashboard</h1>
+
+                <div class="section">
```

Comment:
In React, we want to use `className` instead of `class` because it gives us the option to use variables for class names. So please change all `class=` in this file to `className=`